src/backend/opencl/magma/laset.cpp

```diff
@@ -7,6 +7,50 @@
  * http://arrayfire.com/licenses/BSD-3-Clause
  ********************************************************/
 
+/***********************************************************************
+ * Based on MAGMA library http://icl.cs.utk.edu/magma/
+ * Below is the original copyright.
```

Comment:
I was making it consistent with the remaining magma files (they have the license included in the file). I did not want to remove license for existing files, so just added them everywhere.